### ISSUE TITLE
Email newsletter

### ISSUE THREAD

**Issue by devsgarage** [devsgarage/tulsa-dnug-website#7]
- Link and Blurb
     - http://eepurl.com/csisb
     - Want to stay up with the latest happenings at the Tulsa .NET User Group, sign up for our newsletter and update alerts

**Comment by jeff4intellacore** [comment]
Is there an API for this list manager? Could we do a better integration than just dump onto a signup page?

**Comment by devsgarage** [comment]
Good idea and we will look at adding this to a future iteration. For the time being, we'll shoot for feature parity.

**Comment by jacrys** [comment]
From what I can tell, that is a MailChimp signup form correct?  If so, you can literally create the embed code and dump it:  https://mailchimp.com/help/add-a-signup-form-to-your-website/
